tempesta_fw/http.c

```diff
@@ -748,7 +755,8 @@ tfw_http_adjust_resp(TfwHttpResp *resp, TfwHttpReq *req)
 	if (resp->flags & TFW_HTTP_RESP_STALE) {
 #define S_WARN_110 "Warning: 110 - Response is stale"
 		/* TODO: ajust for #215 */
-		TfwStr wh = {.ptr = S_WARN_110, .len = SLEN(S_WARN_110),.eolen = 2};
+		TfwStr wh = {.data = S_WARN_110, .len = SLEN(S_WARN_110),
+			     .eolen = 2};
```

Comment:
If you're trying to keep a consistent style, then the contents within the curly brackets is separated with spaces. See all other similar constructions above.